src/structures/song_selector.ts

```diff
@@ -191,35 +198,46 @@ export default class SongSelector {
 
     async reloadSongs(
         guildPreference: GuildPreference,
-        isPremium: boolean
+        isPremium: boolean,
+        songList?: MatchedPlaylist
```

Comment:
Doesn't really make sense for a function called `reloadSongs` to take in a list of songs to passthrough?

Reply:
We'll need to retrieve the playlist metadata from the `,spotify` command, so I pass it along here to prevent hitting the endpoint twice